From 91461601268e9a28ff2698afa3978ae8173e62b2 Mon Sep 17 00:00:00 2001
From: ylevavi <yehuda.levavi@tandemg.com>
Date: Wed, 20 Oct 2021 10:37:07 +0300
Subject: [PATCH] [SW-56463] spi: reduce gaudi2 onix baud rate

Reduce SPI BAUD rate so TPM will work properly on ONIX.

Change-Id: Ieef695d1817b6e8a16df94556f024a0865ae03aa
Signed-off-by: ylevavi <yehuda.levavi@tandemg.com>
---

diff --git a/arch/gaudi2/arc/onix.c b/arch/gaudi2/arc/onix.c
index 605400a..0bd31f1 100644
--- a/arch/gaudi2/arc/onix.c
+++ b/arch/gaudi2/arc/onix.c
@@ -21,7 +21,7 @@
 
 u32 hl_spi_baudr_get(void)
 {
-	u32 factor = 2; /* Min Divider */
+	u32 factor = 4; /* Min Divider */
 
 	/* baud rate doesn't support odd number */
 	return (factor + 1) & 0xFFFE;
